js/src/components/paid-ads/budget-section/budget-recommendation/index.js

```diff
@@ -50,21 +33,19 @@ function toRecommendationRange( isMultiple, ...values ) {
 }
 
 const BudgetRecommendation = ( props ) => {
-	const { countryCodes, dailyAverageCost = Infinity } = props;
-	const { data } = useFetchBudgetRecommendationEffect( countryCodes );
-	const map = useCountryKeyNameMap();
-
-	if ( ! data ) {
-		return null;
-	}
+	const {
+		isMultiple,
```

Comment:
I see the `countryCodes` prop was removed but it's still being passed to `BudgetRecommendation` in `js/src/components/paid-ads/budget-section/index.js`